### ISSUE TITLE
Missing properties/methods from header files

### ISSUE THREAD

**Issue by vijaytholpadi** [OmniVirt/OmniVirt-iOS-SDK#5]
Hello,

**What did I attempt:**
Using the framework using both ObjC and Swift projects. Trying both CocoaPods as well as directly importing the library into the project.

**What did I expect to work:**
Able to see the `launchWithContentID` and other methods to use the Library.

**What happened instead:**
The header file generated at:
https://github.com/OmniVirt/OmniVirt-iOS-SDK/blob/master/OmniVirtSDK.framework/Headers/OmniVirtSDK-Swift.h

```
@interface FullscreenVRPlayer : UIViewController
- (void)viewDidLoad;
- (void)dismissViewControllerAnimated:(BOOL)flag completion:(void (^ _Nullable)(void))completion;
@property (nonatomic, readonly) UIInterfaceOrientationMask supportedInterfaceOrientations;
@property (nonatomic, readonly) BOOL shouldAutorotate;
- (nonnull instancetype)initWithNibName:(NSString * _Nullable)nibNameOrNil bundle:(NSBundle * _Nullable)nibBundleOrNil OBJC_DESIGNATED_INITIALIZER;
- (nullable instancetype)initWithCoder:(NSCoder * _Nonnull)aDecoder OBJC_DESIGNATED_INITIALIZER;
@end


SWIFT_CLASS("_TtC11OmniVirtSDK8VRPlayer")
@interface VRPlayer : UIView <WKNavigationDelegate>
- (nonnull instancetype)init SWIFT_UNAVAILABLE;
+ (nonnull instancetype)new SWIFT_DEPRECATED_MSG("-init is unavailable");
- (nullable instancetype)initWithCoder:(NSCoder * _Nonnull)aDecoder OBJC_DESIGNATED_INITIALIZER;
- (void)layoutSubviews;
- (void)webView:(WKWebView * _Nonnull)webView decidePolicyForNavigationAction:(WKNavigationAction * _Nonnull)navigationAction decisionHandler:(void (^ _Nonnull)(WKNavigationActionPolicy))decisionHandler;
- (nonnull instancetype)initWithFrame:(CGRect)frame SWIFT_UNAVAILABLE;
@end
```

Is missing the methods to use the library. It also is missing important properties/methods on `VRPlayer` class too. Thereby making it impossible to use the library

The example apps also do not work with the latest frameworks builds.

**Comment by vijaytholpadi** [comment]
Diving into the git-log shows that these methods were removed in the commit
https://github.com/OmniVirt/OmniVirt-iOS-SDK/commit/c1cad471b4056a6ad64ad978991fa3d9903fe867

Is there a different approach to be taken to use this SDK?

**Comment by vijaytholpadi** [comment]
@nuuneoi @baddth @jasukkas Would you guys happen to have any input on this? Would greatly help.

**Comment by vijaytholpadi** [comment]
This has been fixed in the latest release `1.6.0` 
https://github.com/OmniVirt/OmniVirt-iOS-SDK/releases/tag/v1.6.0
